src/_guides/libraries/library-tour.md

```diff
@@ -1035,6 +1035,38 @@ For more information, see
 [Exceptions](/guides/language/language-tour#exceptions)
 (in the language tour) and the [Exception API reference.][Exception]
 
+### Weak references and finalizers
+Dart is a [garbage-collected][] language,
+which means that any Dart object
+that isn't referenced by at least one other object
```

Comment:
Not sure if it's important, but should this include something like "or from the current scope"?

Reply:
I think that's correct, but there's probably more situations where an object would be kept alive. Maybe we should stick with "that isn't referenced"